server/src/main/java/org/elasticsearch/action/admin/indices/validate/query/TransportValidateQueryAction.java

```diff
@@ -226,4 +245,16 @@ private static String explain(SearchContext context, boolean rewritten) {
             return query.toString();
         }
     }
+
+    private Index[] resolveLocalIndices(ValidateQueryRequest request) {
+        OriginalIndices localIndices = remoteClusterService.groupIndices(request.indicesOptions(), request.indices())
+            .get(RemoteClusterAware.LOCAL_CLUSTER_GROUP_KEY);
+
+        if (localIndices == null) {
+            return Index.EMPTY_ARRAY;
+        }
+
+        // TODO: Need to provide start time?
```

Comment:
Question: do we need to provide the start time to `indexNameExpressionResolver.concreteIndices`?